css-brochure-form/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <form>
+    <h3>LEARN MORE</h3>
+
+    <div class="email">
+      <label for="comp-email" class="email">COMPANY EMAIL
+        <input type="text" id="comp-email" class="email" placeholder="sally@company.co">
+      </label>
```

Comment:
Since you need the label to be indented but you don't want to have the input and select elements indented, making the form control elements a child of the `label` is making this needlessly complicated to style. You should make the `input` and `select` element siblings of their `label` elements, so you can add padding to the `label` without adding padding to the `input` as well.